x-pack/plugins/cases/public/containers/use_get_case_connectors.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useQuery } from '@tanstack/react-query';
+import * as i18n from './translations';
+import { getCaseConnectors } from './api';
+import type { ServerError } from '../types';
+import { casesQueriesKeys } from './constants';
+import { useCasesToast } from '../common/use_cases_toast';
+import type { CaseConnectors } from './types';
+
+// 30 seconds
+const STALE_TIME = 1000 * 30;
```

Comment:
Because two components in the case view page use the same hook I add the stale time to avoid a double fetch on the first render.